.github/workflows/apiBreakTest.yml

```
@@ -0,0 +1,216 @@
+name: Public Interface Breakage Detection
+
+on:
+  pull_request:
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  buildAndCheckAPIBreakage:
```

We've been using `dash-case` for job names.
```suggestion
  build-and-check-api-breakage:
```